internal/provider/resource_model_test.go

```diff
@@ -21,9 +20,7 @@ func TestAcc_ResourceModel(t *testing.T) {
 			{
 				Config: testAccResourceModel(t, modelName, logLevelInfo),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestMatchResourceAttr(
-						"juju_model.model", "name", regexp.MustCompile("^"+modelName+"$"),
-					),
+					resource.TestCheckResourceAttr("juju_model.model", "name", modelName),
```

Comment:
I was wondering if I should do this in my last update... so I'm glad you agree that this is the way :P